apps/cookbook/src/app/examples/toggle-button-example/toggle-button-example.component.html

```diff
@@ -0,0 +1,4 @@
+<kirby-toggle-button [checked]="true" (checkedChange)="onCheckedChange($event)">
```

Comment:
We expose the template through a config object now - e.g. see apps\cookbook\src\app\examples\segmented-control-example\default\default.ts